ifrs17/Utils/ImportCalculationMethods.ipynb

```diff
@@ -356,7 +356,25 @@
         {
             "cell_type": "markdown",
             "source": [
-                "# Helper variables used in calculation",
+                "## Simple Value and Opening Importer"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static HashSet<string> GetAocTypeWithoutCsmSwitch() => new []{AocTypes.BOP, AocTypes.EOP, AocTypes.AM, AocTypes.EA}.ToHashSet();"
```

Comment:
Why a method and not a field?

Reply:
there are many such methods in this notebook. Rather than having them as fields I would pass the storage in (even if not required in the template) such that other implementations could have their own logic.